<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignInForm</class>
 <widget class="QWidget" name="SignInForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="loginwidget" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="UserName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>102</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Password">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>223</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Email">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="EmailEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PhoneNumEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="PhoneNum">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>162</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="createBtn">
   <property name="geometry">
    <rect>
     <x>338</x>
     <y>358</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="backwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>391</width>
     <height>401</height>
    </rect>
   </property>
   <widget class="QPushButton" name="minisizeBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LoginEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 20pt &quot;黑体&quot;;
</string>
    </property>
    <property name="text">
     <string>系统账号登录</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="returnLogBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>348</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>closeBtn</zorder>
   <zorder>minisizeBtn</zorder>
   <zorder>LoginEdit</zorder>
   <zorder>returnLogBtn</zorder>
  </widget>
  <zorder>backwidget</zorder>
  <zorder>loginwidget</zorder>
  <zorder>UserName</zorder>
  <zorder>Password</zorder>
  <zorder>Email</zorder>
  <zorder>EmailEdit</zorder>
  <zorder>PhoneNumEdit</zorder>
  <zorder>PasswordEdit</zorder>
  <zorder>NameEdit</zorder>
  <zorder>PhoneNum</zorder>
  <zorder>createBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
